packages/framework/client-logger/fluid-telemetry/package.json

```diff
@@ -129,15 +129,9 @@
 		"tslib": "^1.10.0",
 		"typescript": "~5.4.5"
 	},
-	"fluidBuild": {
-		"tasks": {
-			"tsc": [
-				"typetests:gen"
-			]
-		}
-	},
 	"typeValidation": {
 		"disabled": true,
-		"broken": {}
+		"broken": {},
+		"entrypoint": "legacy"
```

Comment:
Probably wasn't caught because type-tests are disabled in this package, but this one doesn't have a `./legacy` export. If we enabled them at some point, this would... fail? I think that's fine and at that point we would have to figure out the correct value. Just pointing it out to make sure I'm not missing something about this scenario. Which could potentially be true in other packages where type tests are currently disabled.

cc @seanimam because IIRC we do have an open ADO item to enable type tests in this package.

Reply:
Yeah the generation will fail when this is enabled, but it should be pretty clear in the error message what's wrong. That said I'll fix this instance.

Reply:
```suggestion
		"entrypoint": "public"
```